### DISCUSSION TITLE
GitHub dependency graph integration should support multiple submissions within same workflow job

### DISCUSSION THREAD

**Discussion by michaelbprice** [microsoft/vcpkg#32734] [General]
There are circumstances, such as projects organized as monorepos, where a single workflow job will need to call vcpkg multiple times to complete their build. Currently, the GitHub dependency graph integration does not support multiple invocations from the same workflow job.

**Comment by flichtenheld** [comment]
Just to clarify: You can run separate invocations in the same *workflow*, but not the same *job*. This is because vcpkg always uses the *correlator* value of "<workflowname>-<jobname>" and so multiple invocations in the same job will overwrite each other. Invocations in separate jobs will be treated as separate submissions by GitHub.
